services/processor/native/repository/_Deployments/deploy.go

```diff
@@ -27,15 +28,16 @@ func getInfo(serviceName string) uint32 {
 	return processorType
 }
 
-func getCode(serviceName string) []byte {
```

Comment:
this method is public, so any transaction that attempted to call `getCode(serviceName)` will not be executed properly on the new contract

Reply:
It's a read-only method that is only used by prism.